.github/workflows/nodejs.yml

```diff
@@ -1,73 +0,0 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
-
-name: CI
-
-on:
-  push:
-    branches:
-      - main
-  pull_request:
```

Comment:
we should still run the CI on PRs

Reply:
We discussed that currently cant be done , because of the db migration.